portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,104 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+//HELPER_FUNCTION
+/**Create a typewriter from a given div */
+function typify(div_id){
+  //get the div
+  var div = document.getElementById(div_id);
```

Comment:
let or const https://google.github.io/styleguide/jsguide.html#features-use-const-and-let